osu.Game/Graphics/UserInterface/OsuButton.cs

```diff
@@ -126,15 +125,19 @@ protected override void LoadComplete()
         protected override bool OnClick(ClickEvent e)
         {
             if (Enabled.Value)
-                Background.FlashColour(BackgroundColour.Lighten(0.4f), 200);
+                Background.FlashColour(Color4.White, 800, Easing.OutQuint);
```

Comment:
Having a big worry that this is going to end up making *something* look broken, although I have checked a few places and couldn't find anything, and I'm not about to check every single `OsuButton` or derived in existence...